files/build_templates/docker_image_ctl.j2

```diff
@@ -365,6 +365,8 @@ start() {
         -v mlnx_sdk_ready:/tmp \
         -e SX_API_SOCKET_FILE=/var/run/sx_sdk/sx_api.sock \
         -v /dev/shm:/dev/shm:rw \
+        -v /usr/share/sonic/device/$PLATFORM:/usr/share/sonic/device/$PLATFORM:ro \
+        -v /host/machine.conf:/host/machine.conf:ro \
```

Comment:
@tomer-israel how does this fix the bug? Can you please share some logs before/after?

Reply:
the error on log on 202012 was:

/var/log/syslog:Mar 31 08:10:51.099928 r-anaconda-simx-129 ERR pmon#syseepromd: Failed to load platform-specific eeprom implementation: TypeError("'NoneType' object is not subscriptable")
/var/log/syslog:Mar 31 08:10:52.123772 r-anaconda-simx-129 INFO pmon#supervisord 2021-03-31 08:10:52,120 INFO exited: syseepromd (exit status 5; not expected)

we saw these errors on all simx platform on 202012

the pmon tried to read the file /host/machine.conf 
the python script that tried to read this file was eeprom.py (exist on each mellanix device plugin platform)

`
machine_info = get_machine_info()
onie_platform = machine_info['onie_platform']
if 'simx' in onie_platform:
    platform_path = os.path.join('/usr/share/sonic/device', onie_platform)
    subprocess.check_call(['/usr/bin/xxd', '-r', '-p', 'syseeprom.hex', 'syseeprom.bin'], cwd=platform_path)
    CACHE_FILE = os.path.join(platform_path, 'syseeprom.bin')
`
also this code is trying to read and write on /usr/share/sonic/device/$PLATFORM, so I added it too

Reply:
the cache folder is not part of the pmon mounts (/var/cache/sonic/decode-syseeprom/syseeprom_cache)
we can add it as **read & write** instead of:
`/usr/share/sonic/device/$PLATFORM:/usr/share/sonic/device/$PLATFORM:rw`

if we will choose to mount the cache folder instead of the platform folder we will need to change the eeprom.py file of all the simx platforms to use the cache folder instead so maybe it's better to use the platform folder as I suggested...

what do you think @nazariig ?